docs/contributing/index.rst

```diff
@@ -89,19 +94,132 @@ versions are maintained through the community. Divio provides long term support
 `commercial support <http://divio.ch/en/commercial-support/>`_.
 
 
-*************
+.. _branch_policy:
+
 Branch policy
-*************
+=============
+
+.. versionchanged:: 3.3
+
+We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI.
 
-* **master**: this is the current stable release, the version released on PyPI.
-* **support/3.2.x**: this will be our *next stable release*; this is the most
-  appropriate branch for fixes and patches that will go into the next **master**
-* **develop**: this will be *django CMS 3.3*; this is the most appropriate
-  branch for more substantial features that will need team co-ordination
+    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
+    this branch.
 
-If in doubt, ask on the #django-cms IRC channel on `freenode`_ or the
+``develop``
+    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
+    ``release/x.y.z``.
+
+    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
+    substantial team co-ordination.
```

Comment:
Standard in UK English - co-ordination, co-operation, etc. The cool kids use cördination and cöperation, but I wouldn't have the nerve.